db/db_test.go

```diff
@@ -391,6 +391,12 @@ var _ = Describe("Database operation test", func() {
 					Expect(tx.Commit()).To(Succeed())
 				})
 
+				It("Deletes resources using filter", func() {
+					Expect(tx.DeleteFilter(ctx, serverSchema, transaction.IDFilter(serverResource.ID()))).To(Succeed())
```

Comment:
this test will pass with empty body of DeleteFilter method i.e. it doesn't check if resource from filter were deleted

Reply:
done